portal/static/js/src/modules/TnthAjax.js

```diff
@@ -373,6 +373,12 @@ export default { /*global $ */
                 setTimeout(function() {
                     this.getSubStudyTriggers(userId, params, callback);
                 }.bind(this), 1500*params.retryAttempt);
+                if (params.retryAttempt === params.maxTryAttempts) {
+                    this.postAuditLog(userId, {
+                        context: "access",
```

Comment:
i don't see a great category in the audit/context column, but this isn't a controlled vocab.

perhaps "timeout" or "error" would make more sense.   or stick with "assessment" as that's what is being collected here, so to speak.